docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -0,0 +1,461 @@
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '10.2'
+  - release notes
+labels:
+products:
+  - cloud
+  - enterprise
+  - oss
+title: What's new in Grafana v10.2
+weight: -39
+---
+
+# What’s new in Grafana v10.2
+
+Welcome to Grafana 10.2! Read on to learn about changes to ...
+
+For even more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md). For the specific steps we recommend when you upgrade to v10.2, check out our [Upgrade Guide](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/upgrade-guide/upgrade-v10.2/).
+
+<!-- Template below
+
+> Add on-prem only features here. Features documented in the Cloud What's new will be copied from those release notes.
+
+## Feature
+<!-- Name of contributor -->
+<!-- _[Generally available | Available in private/public preview | Experimental] in Grafana [Open Source, Enterprise]_
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+{{% admonition type="note" %}}
+Use full URLs for links. When linking to versioned docs, replace the version with the version interpolation placeholder (for example, <GRAFANA_VERSION>, <TEMPO_VERSION>, <MIMIR_VERSION>) so the system can determine the correct set of docs to point to. For example, "https://grafana.com/docs/grafana/latest/administration/" becomes "https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/".
+{{% /admonition %}}
+-->
+<!-- Add an image, GIF or video  as below
+
+{{< figure src="/media/docs/grafana/dashboards/WidgetVizSplit.png" max-width="750px" caption="DESCRIPTIVE CAPTION" >}}
+
+Learn how to upload images here: https://grafana.com/docs/writers-toolkit/write/image-guidelines/#where-to-store-media-assets
+-->
+
+## Public dashboards
+
+<!-- Thanos Karachalios -->
+
+_Generally Available in Grafana Open Source and Enterprise_
+
+Public dashboards allow you to share your visualizations and insights to a broader audience without the requirement of a login. You can effortlessly use our current sharing model and create a public dashboard URL to share with anyone using the generated public URL link. To learn more, refer to the [Public dashboards documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/dashboards/dashboard-public/), as well as the following video demo:
+
+{{< video-embed src="/media/docs/grafana/dashboards/public-dashboards-demo.mp4" >}}
+
+## Recorded queries: Record multiple metrics from a single query
+
+<!-- Kyle Brandt, Observability Metrics -->
+
+_Generally available in Grafana Enterprise_
+
+With recorded queries, a single recorded query can now record multiple metrics.
+
+## Dashboards and visualizations
+
+### Generative AI features for dashboards
+
+// Is this actually generally available given steps of needing to also install LLM plugin / have openAI key?
+_Generally available in of Grafana Open Source and Enterprise_
```

Comment:
This should be public preview status 😬

Reply:
Yep, I got that from the Slack thread. It's in my local set of updates.